lib/Common/DataStructures/SizePolicy.cpp

```diff
@@ -9,9 +9,59 @@ static const uint primes[] = {
     1103, 1327, 1597, 1931, 2333, 2801, 3371, 4049, 4861, 5839, 7013, 8419, 10103, 12143, 14591,
     17519, 21023, 25229, 30293, 36353, 43627, 52361, 62851, 75431, 90523, 108631, 130363, 156437,
     187751, 225307, 270371, 324449, 389357, 467237, 560689, 672827, 807403, 968897, 1162687, 1395263,
-    1674319, 2009191, 2411033, 2893249, 3471899, 4166287, 4999559, 5999471, 7199369
+    1674319, 2009191, 2411033, 2893249, 3471899, 4166287, 4999559, 5999471, 7199369, 10453007, 13169977, 20906033
 };
 
+static_assert(_countof(primes) == UNKOWN_MOD_INDEX, "Adding / removing prime numbers ?");
+
+#define MOD_FNC_(x) static uint mod##x(const uint k, const uint) { return k % x##llu; }
```

Comment:
some comments in here RE what you're trying to achieve with the macros will be helpful for new people down the road. e.g., 
```
// If we know the modulo number at compile time, the compiler can optimize instructions.  
// These macros let the compiler do these optimizations for modulus optimizations of known primes
```

Reply:
good one. added!